src/AWSCredentials.jl

```diff
@@ -101,9 +100,54 @@ function AWSCredentials(;profile=nothing)
         println()
     end
 
-    return creds
+    return RenewableAWSCredentials(creds, renew)
+end
+
+@deprecate AWSCredentials(;profile=nothing) get_credentials(RenewableAWSCredentials(profile=profile))
```

Comment:
Since `AWSCredentials` is still a type why do we need to deprecate this?

Reply:
While `AWSCredentials` is still a type we should want to deprecate the constructor that fetches the default credentials since that job has been transferred to `RenewableAWSCredentials`.